base/i18n/base_i18n_export.h

```diff
@@ -4,8 +4,10 @@
 
 #ifndef BASE_I18N_BASE_I18N_EXPORT_H_
 #define BASE_I18N_BASE_I18N_EXPORT_H_
+#include "starboard/configuration.h"
```

Comment:
This looks like it needs its own `#ifdef STARBOARD` bracket around it.

Reply:
added